policy/lib/purl.rego

```diff
@@ -0,0 +1,26 @@
+package lib.purl
+
+import rego.v1
+
+# Extract the repoid
+repoid(purl) := _named_qualifier("repository_id", purl)
+
+#
+# Extract a named quantifier from a purl.
+# (Quantifiers are similar to url params.)
+# Let's fall back to returning null if the quantifier is not present.
```

Comment:
I think this is a bit unusual in rego. Not returning a value is usually what is done in such cases.

Reply:
Yeah, I think you're right. It feels more comfortable to return null, but also less rego-ish.